includes/utilities.php

```diff
@@ -85,3 +85,57 @@ function prefix_path( $url ) {
 
 	return $url;
 }
+
+/**
+ * Add a redirect.
+ *
+ * @param array $redirect Arguments for the redirect.
+ *
+ * @return int The ID of the Post if redirect successfully added. Otherwise returns 0.
```

Comment:
Can we have a better return type than `0`? I know we're using `wp_insert_post()` function signature, and it sucks. I'm fine with `null`, `WP_Error`, or even an exception.